pkg/pushsync/pushsync.go

```diff
@@ -334,35 +292,14 @@ func (ps *PushSync) PushChunkToClosest(ctx context.Context, ch swarm.Chunk) (r *
 			return nil, err
 		}
 
-		rec := &Receipt{
-			Address: swarm.NewAddress(receipt.Address),
-		}
-
-		return rec, nil
+		return &receipt, nil
 	}
 
-	logger.Tracef("pushsync-push: failed to push chunk %s: reached max peers of %v", ch.Address(), maxPeers)
+	logger.Tracef("pushsync: failed to push chunk %s: reached max peers of %v", ch.Address(), maxPeers)
```

Comment:
if this is totally wrong please suggest what is totally correct?
not sure what is the concrete suggestion here.
if we exhausted the aformentioned peer attempt number, then it means we should return a `topology.ErrNotFound` so that the calling context can know that there was no peer found to sync the chunk to.